simulations/uv_simulations.py

```diff
@@ -188,6 +192,9 @@ def to_enu(self, x_ref, y_ref, z_ref):
             x, y, z reference coordinates
         """
         lon_ref, lat_ref = self.to_geodetic(x_ref, y_ref, z_ref, enu=True)
+        if isinstance(x_ref, int):
+            x_ref, y_ref, z_ref = [x_ref], [y_ref], [z_ref]
+            lon_ref, lat_ref = [lon_ref], [lat_ref]
```

Comment:
Warum ist das nötig?

Reply:
This allows to do the calculation also for integer inputs. Before the calculations were only possible when your inputs where arrays of x_ref, y_ref and z_ref values.